### DISCUSSION TITLE
ifstat snmp variant

### DISCUSSION THREAD

**Discussion by vtchelp** [Homebrew/discussions#1179] [Everyday usage]
How can I install ifstat snmp variant?  So I can do 'ifstat -s host' and not get error message 'ifstat: driver snmp not available in this binary!'  

Any help will be appreciated.  Thanks.

**Comment by gromgit** [comment]
From [the project's home page](http://gael.roualland.free.fr/ifstat/):
> If the net-snmp (or ucd-snmp) library is available, ifstat can use it to gather statistics from remote equipments (hosts, routers, switches...) or even the local host if a SNMP daemon is running.

Since there is a `net-snmp` formula available, this is a good opportunity to make your first Homebrew contribution: 

https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request

It _should_ be as simple as adding:
```
depends_on "net-snmp"
```
to the `ifstat` formula, but if there's more to be done, the maintainers would be happy to guide you along.
